### PR TITLE
update dependencies p3

### PR DESCRIPTION
**Pull request by myounus99**

### CODE REVIEW

**Review by jadchaar — APPROVED** [review]
LGTM

**Comment by jadchaar** [comment]
Make sure the dependency changes work with P3 before merging this @myounus99 I see Flask reached v2.0 which has some breaking changes!

File: p3/requirements.txt [hunk 1]
```diff
@@ -1,41 +1,52 @@
...
+isort==5.8.0
+itsdangerous==2.0.0
+Jinja2==3.0.0
+lazy-object-proxy==1.6.0
+MarkupSafe==2.0.0
 mccabe==0.6.1
 more-itertools==8.5.0
 packaging==20.9
+pdbpp==0.10.2
+pkg-resources==0.0.0
```
Comment (jadchaar) [inline p3/requirements.txt:27]: This seems to be breaking P3 install step https://github.com/eecs485staff/p3-insta485-clientside/runs/2604750398.

Comment (awdeorio) [reply p3/requirements.txt:27]: Also, I don't expect pdbpp to be part of the production dependencies.  Did you create a new clean virtual environment before dumping the new deps?

Comment (myounus99) [reply p3/requirements.txt:27]: I had forgotten to do that the first time around, but I redid it just now and it is working locally. I'll push my changes here after getting the P3 repo dependencies pushed

Comment (jadchaar) [reply p3/requirements.txt:27]: Yeah I usually create a clean virtual env before dumping them.

**Pull request merged**